### PR TITLE
Adding Assurance Case 5

### LINKED ISSUE #32: Assurance Case 5

### LINKED ISSUE #40: Assessment of evidence for Assurance case 5
Assess the alignment of the evidence you identified in your diagrams, with that available (or can be made available) from the OSS project. Highlight the gaps.

### PR DESCRIPTION
**Pull request by RocketSockz**

### CODE REVIEW

**Review by Chrs987 — DISMISSED** [review]
Nice clean write up

File: Assurance Cases.md [hunk 1]
```diff
@@ -75,14 +75,18 @@ As for Home Assitant official add-ons, we presume that they are secure and if a
...
-.red[TODO] Insert Assuracne Case 5 Image
+![Assurance Case 1 Diagram:](/images/Assurance_Case_5.PNG)
 
 #### Evidence for Assurance Case 5
+Home Assistant offers users to have access to their Home Assistant integrations outside of the user's local network. This feature is developed by the founders of Home Assistant and serves Home Assistant Cloud (Otherwise known as [Nabu Casa](https://www.nabucasa.com/)) as an optional add-on on top of the traditional Home Assistant integration. The setup process for expanding your integration to allow remote access is rather seamless, with various amounts of [documentation supporting it](https://www.nabucasa.com/config/). To enable the remote access feature, while connected to your Home Assistant locally on a user's network, navigate to the cloud options and remote control is a simple toggle switch, from their Home Assistant takes care of the rest. This feature allows users to enhance their Home Assistant integration with the ability to manage their home outside of their network. However, with the addition of this feature the risk of user's information being hijacked in transit from remote devices to an at home network increases.
```
Comment (jmpinoy) [inline Assurance Cases.md]: The second to last sentence ... navigate to the cloud options and remote control is a simple toggle switch ... reads a little weird

File: Assurance Cases.md [hunk 2]
```diff
@@ -75,14 +75,18 @@ As for Home Assitant official add-ons, we presume that they are secure and if a
...
+
+As for R2, we analze the case in which an attacker obtains the secure certificate by other means. However, as explained in sub-claim C3 it is required that once a connection is established between two machines the local Home Assistant credentials are required in order to allow for [authorization](https://www.nabucasa.com/config/remote/#security). It also a known feature, that Home Assistant allows tracking of all events done on both remote and local sessions allowing user [customized guard locks to be put in place](https://www.home-assistant.io/docs/authentication/). However, the storage of local credentials in Home Assistant is its own caveat that is covered in [Assurance Case 1](#assurance-case-1).
```
Comment (jmpinoy) [inline Assurance Cases.md]: analze --> analyze

**Review by CernickyR — DISMISSED** [review]
Great writeup on Nabu Casa. Links bring a good touch.

File: Assurance Cases.md (hunk 2 above)
Comment (jmpinoy) [reply Assurance Cases.md]: The second sentence ... It also a known feature ... should it be "It's also a known feature"

File: Assurance Cases.md [hunk 3]
```diff
@@ -75,14 +75,18 @@ As for Home Assitant official add-ons, we presume that they are secure and if a
...
+
+As discussed in R3, another possibility is that an attacker is able to generate their own key and attach it to your local Home Assistant instance. However as explained in sub-claim C4, connection certificates are only generated at setup time by Home Assistant or are manually configured on [network](https://www.nabucasa.com/config/remote/#remote-ui). This does bring one glaring worry, however, in the case of Home Assistant manually decrypting traffic while routing it from your remote device, to your Home Assistant integration. This scenario is publically announced, however, and Home Assistant verifies that in no normal circumstance will they read the data in transit. The only situation in which they will decrypt user data is when being forced by a government body, as outline in Undermine UM2 and discussed in the official  [Home Assistant Documentation](https://www.nabucasa.com/config/remote/#security)
```
Comment (jmpinoy) [inline Assurance Cases.md]: The beginning of the last sentence reads a little weird ... This scenario is publically announced, however, and Home Assistant ...

**Review by CernickyR — APPROVED** [review]
g2g on those changes

**Review by Jbibianochavez — APPROVED** [review]
Great write up.

**Pull request merged**